cargo.sh

```diff
@@ -6,6 +6,8 @@
 # This file may not be copied, modified, or distributed except according to
 # those terms.
 
+set -e
```

Comment:
Just to be future-proof against using pipes in the future:

```suggestion
set -eo pipefail
```